pages/faq/index.tsx

```diff
@@ -158,12 +162,7 @@ const FaqPage: NextPage = () => {
   ];
 
   // Separate the selected section
-  const sortedSections = selectedSection
-    ? [
-        ...faqData.sections.filter((section) => section.title === selectedSection),
-        ...faqData.sections.filter((section) => section.title !== selectedSection),
-      ]
-    : faqData.sections;
+  const sortedSections = faqData.sections;
```

Comment:
remove this variable and the comment above

Reply:
Removed